packages/go_router_builder/lib/src/route_config.dart

```diff
@@ -44,7 +44,7 @@ class RouteConfig {
   /// Creates a new [RouteConfig] represented the annotation data in [reader].
   factory RouteConfig.fromAnnotation(
     ConstantReader reader,
-    ClassElement element,
+    InterfaceElement element,
```

Comment:
I was afraid this was breaking, but since ClassElement implements InterfaceElement it looks like we're safe.

Reply:
Yep